src/lwc/rd2ElevateInformation/rd2ElevateInformation.html

```diff
@@ -105,7 +105,7 @@ <h2 slot="title" class="slds-p-top_xx-small" data-qa-locator={qaLocatorHeader}>
                                 </div>
                             </lightning-layout-item>
                         </template>
-                        <template if:true={isEditEnabled}>
+                        <template if:true={isElevateCustomer}>
```

Comment:
I think this conditional render can be completely remove since we already checking if the record is a Elevate record above in line 58

Reply:
Just pushed an update with this change